docs/tips/03-if-else-in-JSX.md

```diff
@@ -33,23 +33,23 @@ That's not valid JS. You probably want to make use of a ternary expression:
 React.render(<div id={condition ? 'msg' : ''}>Hello World!</div>, mountNode);
 ```
 
-If a ternary expression isn't robust enough, you can use `if` statements to determine which
-components should be used.
+You can also use ternary expression to determine which tags or components should be used.
```

Comment:
+1 to keeping `if/else`. I point people coming from traditional templating systems to this page all the time, and `if/else` makes JSX feel more familiar.

If there is a ternary example, it makes sense to keep it within the return statement:

		return (
		  <nav>
		    {isLoggedIn ? <cite>Hello!</cite> : <strong>Log in, please...</strong>}
		    {isLoggedIn ? null : <SignUpButton/>;}
		  </nav>
		);